### PR TITLE
LRQA-70450, LRQA-70451

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-70450
https://issues.liferay.com/browse/LRQA-70451

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Review by weller0246 — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro [hunk 1]
```diff
@@ -599,6 +599,13 @@ definition {
 		}
 	}
 
+	@description = "Go to SAML Admin Portlet"
+	macro openSAMLAdmin {
```
Comment (ferenconodiD) [inline portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:603]: I suggest renaming this macro to goToSAMLAdmin or to something similar, as [openSAML](https://shibboleth.atlassian.net/wiki/spaces/OS30/overview) is an existing implementation of SAML. This name can lead to the misunderstanding, that we actually use openSAML here.

**Review by ferenconodiD — CHANGES_REQUESTED** [review]
Please, see my suggestion!

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

**Comment by Kvale1733** [comment]
@gaborlovasdotcom The SAML admin macro name has been changed.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase [hunk 2]
```diff
@@ -17,6 +17,7 @@ definition {
 
 	@description = "This is a use case for LRQA-64613. View SAML configuration after upgrade 7.3.10"
 	@priority = "5"
+	@uitest
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase:20]: Also need to mark one of ViewSAMLWithEncryptionConfigurationAfterUpgradeXXXX test as uitest too.

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase:20]: I agree

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@gaborlovasdotcom I have added a UI flag to ViewSAMLWithEncryptionConfigurationAfterUpgrade7310

**Comment by Kvale1733** [comment]
ci:test:security

**Comment by Kvale1733** [comment]
ci:stop

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

**Comment by JoyceWang08211** [comment]
LGTM

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**